src/luxor_overrides.jl

```diff
@@ -46,13 +46,80 @@ function setopacity(opacity)
     Luxor.setopacity(current_opacity)
 end
 
+"""
+    fontsize(fsize)
+
+Same as `Luxor.fontsize`: Sets the current font size.
+
+# Example
+```
+fontsize(12)
+text("Hello World!")
+```
+
+# Arguments:
+- `fsize`: the new font size
+"""
 function fontsize(fsize)
     cs  = get_current_setting()
     cs.fontsize  = fsize
     Luxor.fontsize(fsize)
 end
 
+"""
+    get_fontsize(fsize)
+
+Same as `Luxor.get_fontsize` but works with every version of Luxor that is supported by Javis.
+
+# Example
+```
+fontsize(12)
+fsize = get_fontsize()
+text("Hello World! \$fsize")
+```
+
+# Returns
+- `Float64`: the current font size
+"""
 function get_fontsize()
     cs  = get_current_setting()
     return cs.fontsize
 end
+
+"""
+    scale(scl)
+
+Set the scale and multiply it with the current multiplier
+which is i.e. set by [`appear`](@ref) and [`disappear`](@ref).
+
+Normal behavior without any animation is the same as `Luxor.scale`.
+
+# Example
+```
+scale(0.5)
+circle(O, 20, :fill) # the radius would be 10 because of the scaling
+```
```

Comment:
Question: if one were to get the current value of the radius for the circle, would it return the 20 value or the 10 value? Where is the radius scaling getting mutated and stored?

Reply:
The whole canvas for this layer is scaled not the circle itself. So you can scale whatever you wish. The radius stays the same. Similar to doing a Translation.

Reply:
Ah gotcha. This makes sense.